auth/user.go

```diff
@@ -183,6 +183,98 @@ func (user *userImpl) RoleHistory() TimedSetHistory {
 	return user.RoleHistory_
 }
 
+func (user *userImpl) RevokedChannels(since uint64) map[string]uint64 {
+
+	// Grab all channels user has access to
```

Comment:
This comment is probably unnecessary - the call to InheritedChannels() is self-documenting.